src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js

```diff
@@ -15,6 +15,23 @@ pimcore.registerNS('pimcore.plugin.importdefinitions.interpreters.asset_url');
 
 pimcore.plugin.importdefinitions.interpreters.asset_url = Class.create(pimcore.plugin.importdefinitions.interpreters.abstract, {
     getLayout : function (fromColumn, toColumn, record, config) {
+        var deduplicateByUrlEnabled = Ext.isDefined(config.deduplicate_by_url) ? config.deduplicate_by_url : false
```

Comment:
I'm not in love with this code, TBH, it looks quite verbose for what it does.

Reply:
yep.... nothing we can do about it.